### ISSUE TITLE
No labels for regional (?) bus lines

### ISSUE THREAD

**Issue by charraeus** [wiesty/MMM-MVVWiesty#1]
Hi Wiesty, great that you set up an alternative to the mvvmunich-module which seems not be maintained anymore.

I live in Putzbrunn where are many bus lines with a 3-digit-line number. For these lines no bus-icon is displayed. If you want to test use this station-id: de:09184:2172

Liebe Grüße, Christian

**Comment by wiesty** [comment]
Sure, will add that!

**Comment by wiesty** [comment]
Added Icons with commit  [8b45b53cd4de25d5d2c197db8dd48612ed9cce12](https://github.com/wiesty/MMM-MVVWiesty/commit/8b45b53cd4de25d5d2c197db8dd48612ed9cce12)

**Comment by charraeus** [comment]
Thank you for the icons. Is it possible to also left-align the icons? That would look better. See attached screenshot.


![IMG_4949](https://github.com/wiesty/MMM-MVVWiesty/assets/38609969/f3947f5d-7efb-4104-8421-f0ee466b3a3c)

**Comment by wiesty** [comment]
> Thank you for the icons. Is it possible to also left-align the icons? That would look better. See attached screenshot.
> 
> 
> 
> 
> 
> ![IMG_4949](https://github.com/wiesty/MMM-MVVWiesty/assets/38609969/f3947f5d-7efb-4104-8421-f0ee466b3a3c)
> 
> 
> 
> 
> 
> 

Whoops! Sure totally forgot that. I only developed it for the left side 😂 yes will do!

**Comment by wiesty** [comment]
> Thank you for the icons. Is it possible to also left-align the icons? That would look better. See attached screenshot.
> 
> ![IMG_4949](https://user-images.githubusercontent.com/38609969/255324293-f3947f5d-7efb-4104-8421-f0ee466b3a3c.jpeg)

Kindly check again. Should be centered now.

**Comment by charraeus** [comment]
Now all fields are centered... see attached picture.
From my point of view all fields should be **right-aligned if on the right side** and centered if the module is in the center-column of MagicMirror. 
Maybe you should add an additional column to your module as first column of the table which contains the icons?

![IMG_4957](https://github.com/wiesty/MMM-MVVWiesty/assets/38609969/37d3a83d-92b8-49d9-803b-b23e1b001e27)

**Comment by wiesty** [comment]
Sure @charraeus , would you mind sharing your css if you have a custom css? Which image are you using Docker or Ubuntu itself? On my Magic mirror Image it looks the same no matter where I am placing it via config.

**Comment by wiesty** [comment]
Check again @charraeus, 
as I am not able to reproduce your error I reverted the changes and added a css class to the td element called "icon-cell" 
Please update the module again, if there is still the error, use the custom.css in the css folder from your magic mirror to modify the icon cell td to your needs :)

**Comment by charraeus** [comment]
Hi @wiesty , I did not change anything - just made a `git pull` and restarted magicmirror.
I use magicmirror on Raspberry Pi with Raspian in the current version. No Docker, no Ubuntu.

Just pulled the latest version, which leads to this result, which imho is ok. Thank's a lot for your effort :-) 
![IMG_4959](https://github.com/wiesty/MMM-MVVWiesty/assets/38609969/3c5e08b0-0fcc-4f00-847a-7c2e3a67d689)

**Comment by wiesty** [comment]
Good to know.
Great that this solution worked for you :) @charraeus 
Cheers

**Comment by charraeus** [comment]
Sorry to bother you again: With the latest MM-Version (and the latest MMM-MVVWiesty) all icons are gone again. Tested with ".mvv-table .icon-cell" text-align left, center and right. No change at all. 

![screenshot](https://github.com/wiesty/MMM-MVVWiesty/assets/38609969/9bb5d733-7e8d-46ba-90e3-34fb5d7abe2d)

**Comment by wiesty** [comment]
No problem, will fix this within 2 weeks :)

**Comment by wiesty** [comment]
Hi,
I updated my MM to your version and I could not reproduce your error, it is working perfectly with icons but I will have closer look to your issue in the following week.

**Comment by wiesty** [comment]
Please reinstall the module folder again. Check if you have the latest version by adding a custom header. (Like your last issue) 

If the error is still there maybe you could check the magic mirror page on a computer and reading the console log by pressing F12 in your browser. Let me know if there are any errors :)

**Comment by charraeus** [comment]
I've done the following:
```
cd Magicmirror/modules
rm -rf MMM-MVVWiesty
git clone https://github.com/wiesty/MMM-MVVWiesty.git
cd ..
npm install node-fetch
```

Unfortunately neither the icons nor the custom header is shown. At the weekend I'll try to read the console log and give feedback.


Meanwhile, this is my config (the line numbers are not part of the config file):

```
380             module: "MMM-MVVWiesty",                                                                                
381             disabled: false,                                                                                        
382             position: "bottom_right",                                                                               
383             header: "MVG Putzbrunn, Michael-Haslbeck-Straße",                                                       
384             config: {                                                                                               
385                 updateInterval: 60000, // Aktualisierungsintervall in Millisekunden (hier: 1 Minute)                
386                 maxEntries: 4, // Maximale Anzahl der angezeigten Einträge in der Tabelle                           
387                 stopId: "de:09184:2172", // Stop-ID im Format de:09162:6                                            
388                 filter: {                                                                                           
389                     "55": "Ostbahnhof",                                                                             
390                     "212": "Neuperlach Süd (U) (S)"                                                                 
391                 }                                                                                                   
392             }                                                                                                       
393         },                                                                                                          
394         {                                                                                                           
395             module: "MMM-MVVWiesty",                                                                                
396             disabled: false,                                                                                        
397             position: "bottom_right",                                                                               
398             header: "MVG Putzbrunn, Kirchenzentrum",                                                                
399             config: {                                                                                               
400                 updateInterval: 60000, // Aktualisierungsintervall in Millisekunden (hier: 1 Minute)                
401                 maxEntries: 5, // Maximale Anzahl der angezeigten Einträge in der Tabelle                           
402                 stopId: "de:09184:2176", // Stop-ID im Format de:09162:6                                            
403                 filter: {                                                                                           
404                     "X204": "München, St. Achaz ü. Unterhaching (S)",                                               
405                     "211W": "",                                                                                     
406                     "211": "Neuperlach Süd (U) (S)",                                                                
407                     "453": "Neuperlach Süd (U) (S)"
408                 }
409             }
410         },
411  
```

**Comment by wiesty** [comment]
May I ask which image you are using for MagicMirror? Is it a DockerImage or a VM?

**Comment by charraeus** [comment]
I did the installation with the installation script that is linked on the Magicmirror web site.Best regards,ChristianVon meinem Händi gesendetAm 21.02.2024 um 14:42 schrieb Wiesty ***@***.***>:﻿
May I ask which image you are using for MagicMirror? Is it a DockerImage or a VM?

**Comment by wiesty** [comment]
I have the image ich777/magic-mirror2. I will try to reproduce your error by setting up a VM someday. After that, I could reinvestigate if I would get the same issue.

**Comment by luaxv0** [comment]
@charraeus I found another transport type, repull the module. It is fixed for me! 👍🏻

**Comment by charraeus** [comment]
Hi Whiesty, - it's me again. 

- Again no logos for busses with numbers like "X204" or "212". :-(
- Sometimes no times are displayed (see bus 212 in screenshot)
- And the header is not displayed

No error messages in the magicmirror-logs.

I did a full new setup with a docker-installation based on the image "karsten13/magicmirror:alpine". Magicmirror runs in server-mode and I use Safari to connect to the MagicMirror.

**Part of my config.js**
```
		{
			module: "MMM-MVVWiesty",
			disabled: false,
			position: "bottom_right",
			header: "MVG Putzbrunn, Michael-Haslbeck-Straße",
			config: {
				updateInterval: 60000, // Aktualisierungsintervall in Millisekunden (hier: 1 Minute)
				maxEntries: 4, // Maximale Anzahl der angezeigten Einträge in der Tabelle
				stopId: "de:09184:2172", // Stop-ID im Format de:09162:6
				filter: { }
			}
		},
		{
			module: "MMM-MVVWiesty",
			disabled: false,
			position: "bottom_right",
			header: "MVG Putzbrunn, Kirchenzentrum",
			config: {
				updateInterval: 60000, // Aktualisierungsintervall in Millisekunden (hier: 1 Minute)
				maxEntries: 5, // Maximale Anzahl der angezeigten Einträge in der Tabelle
				stopId: "de:09184:2176", // Stop-ID im Format de:09162:6
				filter: { 
					"X204": "München, St. Achaz ü. Unterhaching (S)",
					"211W": "",
					"211": "Neuperlach Süd (U) (S)",
					"212": "Neuperlach Süd (U) (S)",
					"453": "Neuperlach Süd (U) (S)",
					"455": "Antholing"
				}
			}
		},

```

**log and system information**
```
[2024-06-08 15:23:57.637] [LOG]   Starting MagicMirror: v2.27.0 
[2024-06-08 15:23:57.646] [LOG]   Loading config ... 
[2024-06-08 15:23:57.647] [LOG]   config template file not exists, no envsubst 
[2024-06-08 15:23:57.715] [LOG]   Loading module helpers ... 
[2024-06-08 15:23:57.716] [LOG]   No helper found for module: alert. 
[2024-06-08 15:23:57.794] [LOG]   Initializing new module helper ... 
[2024-06-08 15:23:57.795] [LOG]   Module helper loaded: updatenotification 
[2024-06-08 15:23:57.796] [LOG]   No helper found for module: clock. 
[2024-06-08 15:23:57.796] [LOG]   No helper found for module: weather. 
[2024-06-08 15:23:57.834] [LOG]   Initializing new module helper ... 
[2024-06-08 15:23:57.834] [LOG]   Module helper loaded: newsfeed 
[2024-06-08 15:23:57.881] [LOG]   Initializing new module helper ... 
[2024-06-08 15:23:57.881] [LOG]   Module helper loaded: calendar 
[2024-06-08 15:23:57.884] [LOG]   Initializing new module helper ... 
[2024-06-08 15:23:57.884] [LOG]   Module helper loaded: MMM-MoonPhase 
[2024-06-08 15:23:57.884] [LOG]   No helper found for module: MMM-MVVWiesty. 
[2024-06-08 15:23:57.884] [LOG]   All module helpers loaded. 
[2024-06-08 15:23:57.887] [LOG]   Starting server on port 8080 ...  
[2024-06-08 15:23:57.892] [LOG]   Server started ... 
[2024-06-08 15:23:57.893] [LOG]   Connecting socket for: updatenotification 
[2024-06-08 15:23:57.893] [LOG]   Starting module helper: updatenotification 
[2024-06-08 15:23:57.893] [LOG]   Connecting socket for: newsfeed 
[2024-06-08 15:23:57.893] [LOG]   Starting node helper for: newsfeed 
[2024-06-08 15:23:57.893] [LOG]   Connecting socket for: calendar 
[2024-06-08 15:23:57.893] [LOG]   Starting node helper for: calendar 
[2024-06-08 15:23:57.893] [LOG]   Connecting socket for: MMM-MoonPhase 
[2024-06-08 15:23:57.893] [LOG]   Starting module helper: MMM-MoonPhase 
[2024-06-08 15:23:57.893] [LOG]   Sockets connected & modules started ... 
[2024-06-08 15:23:57.894] [INFO]  
>>>   Ready to go! Please point your browser to: http://0.0.0.0:8080   <<< 
[2024-06-08 15:23:58.148] [INFO]  System information:
### SYSTEM:   manufacturer: Gigabyte Technology Co., Ltd.; model: Docker Container; raspberry: undefined; virtual: false
### OS:       platform: linux; distro: Alpine Linux; release: v3.20; arch: x64; kernel: 6.1.79-Unraid
### VERSIONS: electron: undefined; used node: 20.14.0; installed node: 20.14.0; npm: 10.7.0; pm2: 
### OTHER:    timeZone: Europe/Berlin; ELECTRON_ENABLE_GPU: undefined 
```

![IMG_6075](https://github.com/wiesty/MMM-MVVWiesty/assets/38609969/bc50ff53-d34f-4d3d-95bd-e95b9fe0599f)

**Comment by wiesty** [comment]
Will check!

**Comment by wiesty** [comment]
Fixed with Commit [6ef21457274e25b07577e6fd17c79ffe4b0d25f2](https://github.com/wiesty/MMM-MVVWiesty/commit/6ef21457274e25b07577e6fd17c79ffe4b0d25f2)

@charraeus I tested your configuration and was able to reproduce your error. Please pull the latest version. It is fixed now.
